client/app/billing/autoRenew/bulk/bulk.controller.js

```diff
@@ -0,0 +1,16 @@
+export default class {
+  $onInit() {
+    this.expand = {};
+
+    this.availableServices = this.services[undefined];
+    this.unavailableServices = _.omit(this.services, undefined);
```

Comment:
```suggestion
    this.unavailableServices = _.pickBy(this.services, service => service);
```

(`_.omit` will be removed in `Lodash 5`)

Reply:
Nevermind, you use `Lodash 3`, but keep it in mind :)